extensions/README.md

```diff
@@ -136,3 +136,8 @@ These files should be kept together in order to preserve relative `$ref` links.
 See the [EU](eo/) extension file structure as an example.
 * Specification examples should be stored in an `examples` directory. 
 * The specification schema file(s) should be stored in a `json-schema` directory. 
+
+Make sure to choose a meaningful name for the extension folder as it will be the shortcut
```

Comment:
If we go the object route for defining extensions, the key can be used as the shortcut for identifying the extension schema.